CharacterCreation/Class-Specific Documentation/Doctor/Doctor Malpractice Procedures.txt

```diff
@@ -543,280 +497,4 @@ Fortitude is permanently reduced by 2. You gain 2 Contagions.
 
     Their Lymph Node: The target loses their lymph node. They lose 40 health and
 up to 120 Nanites. This ability will not drop a target's nanite count below 0. 
```

Comment:
Oh no! We missed a reference to old nanites!

Reply:
THROW IT IN THE FIRE!!!

Reply:
Into the fire it has been thrown.